apps/coscrad-frontend/src/components/AllResources/ResourceInfoPresenter/ResourceInfo.presenter.tsx

```diff
@@ -8,15 +9,16 @@ export const ResourceInfoPresenter = ({
     label,
     schema,
     link: apiIndexRoute,
-}: IAggregateInfo): JSX.Element => {
-    const resourceIndexLink =
-        resourceType !== 'term' ? (
-            <Link to="/ResourceIndex" state={{ schema, data: resourceType, link: apiIndexRoute }}>
-                View Resources of type {label}
-            </Link>
-        ) : (
-            <Link to="/Resources/Terms">View Terms</Link>
-        );
+}: IAggregateInfo<ResourceType>): JSX.Element => {
+    const resourceIndexLink = (
+        [ResourceType.term, ResourceType.photograph] as ResourceType[]
```

Comment:
I suppose we are trying to strangle out the old dynamic index component. Be sure to remove this once the list is complete,